### ISSUE TITLE
Docker image not working

### ISSUE THREAD

**Issue by jonatasbaldin** [apiaryio/apiary-client#145]
Hi, how are you doing?

The `apiaryio/client` Docker image is not working when using the `preview` command.

Here some logs:
```
$ docker run apiaryio/client preview
No API Description Document found
```

When I use with the `--path` argument, I got errors about an invalid path:
```
$ docker run apiaryio/client preview --path="/path/to/docs/api.apib"
Invalid path /path/to/docs/docs/api.apib
```

With the `gem`, however, it works flawlessly. 

Docker info:
```
$ docker -v
Docker version 1.12.5, build 7392c3b
```

The image used is the `latest` to this date.

If you need any more information, please let me know!

**Comment by tu1ly** [comment]
@jonatasbaldin if client is running inside the docker it is not possible to access host filesystem directly - it has to be mounted https://docs.docker.com/engine/tutorials/dockervolumes/ at first. However it won't work anyway because of missing web browser in the image. We'll fix this soon.

**Comment by nadade** [comment]
@jonatasbaldin just to add to @tu1ly's response, you will also need to enable the `server` mode, i.e. something like this:

``docker run --volume <local-path-to-apib>:/tmp -P --net=host apiaryio/client preview --path="/tmp/<path-to-apib>" --server``

When you run this, you will be able to access your documentation at `http://localhost:8080`.

Let me know if this works for you.

**Comment by jonatasbaldin** [comment]
Hi @nadade, thanks for the comment! Unfortunately I could not run using your command, but I adapted it to get it working:

`docker run --volume ~/github/singularity-directory-backend/docs:/tmp -p 8080:8080 apiaryio/client preview --path="/tmp/api.apib" --server --host=0.0.0.0`

I needed to drop the `--net=host` and publish the port using `-p 8080:8080`. I also added `--host=0.0.0.0` because the Apiary server was listening just on 127.0.0.1 and I couldn't access from my localhost.

Anyway, it's working now, thank you all for the support!
